src/app/services/data/trainrun.service.ts

```diff
@@ -799,6 +800,10 @@ export class TrainrunService {
   public getBackwardNonStopIterator(node: Node, trainrunSection: TrainrunSection) {
     return new BackwardNonStopTrainrunIterator(this.logService, node, trainrunSection);
   }
+  
+  public getExpandedIterator(node: Node, trainrunSection: TrainrunSection) {
```

Comment:
I'm not sure this function is used anymore

Reply:
True, good catch i'll remove it